p2p/transport/webrtc/transport_test.go

```diff
@@ -17,6 +18,7 @@ import (
 	quicproxy "github.com/quic-go/quic-go/integrationtests/tools/proxy"
 
 	"golang.org/x/crypto/sha3"
+	"golang.org/x/exp/rand"
```

Comment:
Is this the rand you want to use?

Reply:
I did mostly for this: https://github.com/golang/go/issues/21835

But looks like some of the changes have made it to math/rand and go1.23 will have a migrate tool to move to math/rand/v2. So the stdlib rand is fine here.